conference/SE/icse.yml

```diff
@@ -4,12 +4,11 @@
   rank: A
   dblp: icse
   confs:
-    - year: 2022
-      id: icse22
-      link: https://conf.researchr.org/home/icse-2022
+    - year: 2023
```

Comment:
Please keep the history instead of edit it, like this:
https://github.com/ccfddl/ccf-deadlines/blob/aae68886d4991d9648683603e0cf5e8ef9f02673/conference/DB/sigmod.yml#L6-L26

Reply:
Ok, thanks for your advice!